src/lib/account.js

```diff
@@ -0,0 +1,18 @@
+const dotProp = require('dot-prop')
+
+const supportsBooleanCapability = (account, capability) => {
```

Comment:
A non boolean capability is build minuets (for example)

Reply:
Should this be added as a code comment?